lib/src/users.rs

```diff
@@ -144,6 +148,7 @@ impl UserConfig {
                 email: row.get(3)?,
                 github: github_name.clone(),
                 mute_direct_messages: db::to_bool(row.get(4)?),
+                muted_repos: row.get::<usize, String>(6)?.split(",").map(|s| s.to_owned()).collect(),
```

Comment:
It turns out `cargo test` wasn't running the tests in this file like I thought it was. When I ran them specifically I found the tests were failing and I tracked it down to this line, but I don't have a great way of debugging it. Any ideas why this is causing the lookup to fail?

Reply:
hmm. not sure why cargo test wouldn't be running this one. 🤔

Reply:
i think it probably depends on the error message is. will take a look.

Reply:
```suggestion
                muted_repos: row.get::<usize, String>(5)?.split(",").map(|s| s.to_owned()).collect(),
```